### ISSUE TITLE
Vue packages version mismatch

### ISSUE THREAD

**Issue by sebastiansulinski** [pinguinjkeke/vue-local-storage#43]
Can we update `vue` dependency so that it works with latest release please?

```bash
Module build failed: Error: 

Vue packages version mismatch:

- vue@2.5.15
- vue-template-compiler@2.5.13
```

**Comment by pinguinjkeke** [comment]
Hello! Thank you for opening issue.
It's fixed in just released 0.6.2 version.

**Comment by sebastiansulinski** [comment]
Great - thanks @pinguinjkeke
